cloudhub-2/modules/ROOT/pages/ch2-config-endpoints-paths.adoc

```diff
@@ -154,7 +154,13 @@ An application communicates with another application via:
 
 If you use the internal DNS name, traffic remains within the private space network. You can delete or omit the externally exposed endpoint when deploying an application to a private space. In that case, you can use the application's internal endpoint for internal traffic.
 
-If you use the cluster local endpoint, the traffic doesn't leave the cluster. However, the cluster local endpoint isn't highly available. During some cluster operations such as disaster recovery, the endpoint can be unreachable. The cluster local endpoint allows traffic within the same environment only.
+If you use the cluster local endpoint, the traffic doesn't leave the cluster. However, the cluster local endpoint isn't highly available. During some cluster operations such as disaster recovery, the endpoint can be unreachable. 
+
+During disaster recovery, the cluster local DNS endpoint can fail if an application has migrated to a new cluster. This is because traffic routed through the local DNS endpoint doesn't traverse the ingress load balancer, which is essential for directing traffic to the new cluster. However, using other available endpoints forces  traffic to pass through the ingress load balancer. The ingress load balancer finds out if a disaster recovery event is happening. It also knows if an application linked to a certain endpoint has been moved to a new cluster. It then routes traffic to the new cluster if needed.
```

Comment:
```suggestion
During disaster recovery, the cluster local DNS endpoint can fail if an application migrates to a new cluster. This is because traffic routed through the local DNS endpoint doesn't traverse the ingress load balancer, which is essential for directing traffic to the new cluster. However, using other available endpoints forces traffic to pass through the ingress load balancer. The ingress load balancer detects if a disaster recovery event is happening. It also knows if an application linked to a certain endpoint has moved to a new cluster. It then routes traffic to the new cluster if needed.
```